src/modules/Home/Home.js

```diff
@@ -48,21 +42,21 @@ function Home() {
   let genreDOMElements = [];
   genreData.forEach(genre => {
     genreDOMElements.push(
-      <div className="genre">
+      <div className={`${styles["genre"]}`}>
```

Comment:
This is different from what I did when using a CSS Module for the Song page. Should I change the Song page to match the formatting that you've done?